consent-ui/package.json

```diff
@@ -25,7 +25,8 @@
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
     "bootstrap": "^4.4.1",
-    "rxjs": "~6.4.0",
+    "ng-http-loader": "^6.0.1",
```

Comment:
In consent-ui we have angular 8, according to docs:
https://www.npmjs.com/package/ng-http-loader
shouldn't we use 7+ version of library

Reply:
I tried version 7 but there is a compatibility issue, therefore, I downgraded to v6.
I seems like v7 just work on angular 9